<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>ReferencesPlugin::DialogReferences</class>
 <widget class="QDialog" name="ReferencesPlugin::DialogReferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>References Search</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find References To This Address:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtAddress">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSkipNoAccess">
     <property name="text">
      <string>Skip Regions With No Access Rights</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtAddress</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>chkSkipNoAccess</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReferencesPlugin::DialogReferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReferencesPlugin::DialogReferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
